src/components/common/form/RadioButtonGroup.tsx

```diff
@@ -36,21 +36,28 @@ export default function RadioButtonGroup({
   muiRadioButtonGridProps,
 }: RadioButtonGroup) {
   const { t } = useTranslation('one-time-donation')
-  const [field, meta, helpers] = useField(name)
+  const [field, meta, { setValue }] = useField(name)
   const helperText = meta.touched ? translateError(meta.error as TranslatableField, t) : ''
-  const { setValue } = helpers
   return (
-    <FormControl required component="fieldset" error={Boolean(meta.error) && Boolean(meta.touched)}>
-      <RadioGroup value={field.value} name={name} {...muiRadioGroupProps}>
+    <FormControl
+      fullWidth
+      required
+      component="fieldset"
+      error={Boolean(meta.error) && Boolean(meta.touched)}>
+      <RadioGroup
+        onChange={(e, v) => {
```

Comment:
Great touch for moving the change handler to the radio group instead of individual buttons